src/apis/shops.ts

```
@@ -1,12 +1,58 @@
+    return response;
   } catch (err) {
     throw err;
   }
+}
+
+export const productPurchase = async () => {
+  try {
+    const response = await instance.post(`${shops}`);
+    return response;
+  } catch (err) {
+    throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)